script.js

```diff
@@ -0,0 +1,232 @@
+const datos = [
+    {id: 9,cat: 'General Knowledge'},
+    {id: 10,cat: 'Entertainment: Books'},
+    {id: 11,cat: 'Entertainment: Film'},
+    {id: 12,cat: 'Entertainment: Music'},
+    {id: 13,cat: 'Entertainment: Musicals & Theatres'},
+    {id: 14,cat: 'Entertainment: Television'},
+    {id: 15,cat: 'Entertainment: Video Games'},
+    {id: 16,cat: 'Entertainment: Board Games'},
+    {id: 17,cat: 'Science & Nature'},
+    {id: 18,cat: 'Science: Computers'},
+    {id: 19,cat: 'Science: Mathematics'},
+    {id: 20,cat: 'Mythology'},
+    {id: 21,cat: 'Sports'},
+    {id: 22,cat: 'Geography'},
+    {id: 23,cat: 'History'},
+    {id: 24,cat: 'Politics'},
+    {id: 25,cat: 'Art'},
+    {id: 26,cat: 'Celebrities'},
+    {id: 27,cat: 'Animals'},
+    {id: 28,cat: 'Vehicles'},
+    {id: 29,cat: 'Entertainment: Comics'},
+    {id: 30,cat: 'Science: Gadgets'},
+    {id: 31,cat: 'Entertainment: Japanese Anime & Manga'},
+    {id: 32,cat: 'Entertainment: Cartoon & Animations'},  
+    ];
```

Comment:
Great job putting your logic into data... it's a great principle to follow... imagine if you had to do an `if`/`else-if` chain or a `switch` statement for this?!

One way to improve this would be to make `datos` an object, where each key is the category name and each value the matching `id`. That way, you could skip your `.find'-using function` and use the `datos` object with bracket notation: `catId = datos[categoria]` would get you the id! The object would be easier to read and maintain, and is simply an evolution of what you're doing already.

Keep pushing yourself and experimenting!